src/promptflow/promptflow/_sdk/operations/_run_submitter.py

```diff
@@ -295,16 +296,27 @@ def _submit_bulk_run(self, flow: Flow, run: Run, local_storage: LocalStorageOper
         run._dump()  # pylint: disable=protected-access
         try:
             bulk_result = flow_executor.exec_bulk(mapped_inputs, run_id=run_id)
+            # Filter the failed line result
+            failed_line_result = next(
+                filter(lambda result: result.run_info.status == Status.Failed, bulk_result.line_results), None)
+            if failed_line_result:
+                raise UserErrorException(**failed_line_result.run_info.error)
```

Comment:
What will this run status be if error raised here? I don't see the the status set clause in `except` section?

Reply:
revert it, only log warning msg and not change batch run status to failed